src/SubHask/Category.hs

```diff
@@ -130,7 +130,7 @@ instance Category (->) where
     type ValidCategory (->) (a :: *) = ()
     id = P.id
 
-    {-# NOINLINE (.) #-}
+    {-# INLINE (.) #-}
```

Comment:
This should be marked INLINE.  I believe I changed this for debugging purposes and forgot to change it back.  Good catch.